releasenotes/notes/add-component-tool-ffe9f9911ea055a6.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+ - |
+   Introduced the ComponentTool, a new tool that wraps Haystack components allowing them to be utilized as tools for LLMs (various ChatGenerators). This tool supports automatic function schema generation, input type validation, and offers enhanced integration with various data types.
```

Comment:
You mentioned limitations in the call regarding which components are supported. Could you please add them to the release note? If there is a particular Haystack component that is not supported mention it here as an example.